package.json

```diff
@@ -36,7 +36,7 @@
     "@edx/brand": "npm:@edx/brand-openedx@1.1.0",
     "@edx/frontend-build": "5.6.14",
     "@edx/frontend-platform": "1.14.9",
-    "@edx/paragon": "12.8.0",
+    "@edx/paragon": "16.3.2",
```

Comment:
[sanity check] Have we looked through the paragon release notes for versions 13, 14, 15,and 16 to ensure that they don't include any breaking changes that affect this repo?

Reply:
Yes I've looked into the release notes and also tested by running it locally. The header loads perfectly with paragon v16.